docs/scripts/data/constants.ts

```diff
@@ -1,3 +1,3 @@
-export const PROMISE_POOL_CONCURRENCY = 10;
+export const PROMISE_POOL_CONCURRENCY = 5;
```

Comment:
Fix the memory leak. 

Since the Github Action Step didn't change after reducing the `PROMISE_POOL_CONCURRENCY` to 5, we don't have to test numbers bigger than 5 to reduce the testing time.

Reply:
Technically, this isn't a memory leak but is an optimization, right?